test/conftest.py

```diff
@@ -1,6 +1,7 @@
 import pytest
 import torch
 from multiprocessing import Process
+import builtins
```

Comment:
Could you add a skip if to the tfe tests if keras is not available?

Reply:
and what would be the condition of the skipif?

Reply:
if not depency_check 🙂